packages/core/src/shared/types.ts

```diff
@@ -0,0 +1,16 @@
+import { GraphQLResolveInfo } from 'graphql';
+
+export type ID = string;
+export type Nil = undefined | null;
+export type Maybe<T> = T | Nil;
+export type Plural<T> = T | T[];
+export type Single<T> = T extends Array<infer R> ? R : T;
+export type ResolveFn<TContext> = (
+  parent: any,
+  args: Record<string, any>,
+  context: TContext,
+  info: GraphQLResolveInfo
+) => any;
+
+// Kamil: somehow our build process doesn't emit `types.d.ts` file, this should force it...
+export {};
```

Comment:
Maybe create an issue in `bob` and add a link to it here?

Reply:
it's related to `include` in tsconfig, it should be solved now. In Bob, when you use `files: []` it will be dropped by typescript plugin of rollup